xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/javascript.vm

```diff
@@ -85,6 +85,7 @@
     'drawer': "#removeJsSuffix($services.webjars.url('drawer', ""js/drawer${jsExtension}""))",
     'deferred': $xwiki.getSkinFile('uicomponents/require/deferred.js'),
     'moment-jdateformatparser': $services.webjars.url('moment-jdateformatparser', 'moment-jdateformatparser.min'),
+    'moment-timezone': $services.webjars.url('org.webjars.npm:moment-timezone', 'builds/moment-timezone-with-data.min'),
```

Comment:
hmm does it mean we have a hardcoded dependency in flamingo. Imagine that the lightbox modules were in xwiki-contrib, how would you have done it?

Reply:
small open question: what id format should we use for paths? I see some use camel case while others don't. `moment-timezone` or `moment-timeZone`. Same question for `moment-jdateformatparser`.

Reply:
I am not sure what you mean by hardcoded.
If it were in contrib, then I would have added a separated `require.config({ paths: {'moment-timezone':  ...} })` on my script. But here I added it to the require package, just like it's done for `moment-jdateformatparser`